packages/core/strapi/lib/services/entity-service/types/index.d.ts

```diff
@@ -1,6 +1,20 @@
+import type { Attribute, Common, EntityService } from '@strapi/strapi';
+
 // TODO: Move params to @strapi/utils (related to convert-query-params)
 export * as Params from './params';
 
 export * from './plugin';
 
-export interface EntityService {}
+type Pagination = {
```

Comment:
Does moving this outside make sense?

Reply:
it does but since there is already a task for return types I wanted to think about this on a later stage, this is more of a working placeholder

Reply:
Actually, There is already a Pagination type

Reply:
> Actually, There is already a Pagination type

Not exactly the same since it relates to params

However, it may be better to create a type in a file like types/data.d.ts. Maybe something like "Paginated<TResponse>"  that would add the pagination object on the side. What do you think?